raiden/tests/utils/transport.py

```diff
@@ -80,7 +80,7 @@ def setup_broadcast_room(servers: List["ParsedURL"], broadcast_room_name: str) -
             "Leaving the room failed. This is done otherwise there would be "
             "a ghost user in the broadcast room"
         )
-        assert room.leave(), msg
+        assert room.leave() is None, msg
```

Comment:
Since `room.leave()` now raises exceptions this could be changed to a `try: except:` instead (since the negative case can never happen anymore).

But we can do that in a follow up PR. Just remember to make an issue for it :)

Reply:
#5596 also mentions proper handling of leave_room method, i can add it there